packages/core/logging/core-logging-server-internal/src/logging_system.ts

```diff
@@ -110,6 +115,27 @@ export class LoggingSystem implements ILoggingSystem {
     }
   }
 
+  /**
+   * A mechanism for specifying a set of "global" {@link LogMeta} that we want
+   * to inject into all log entries.
+   *
+   * @example
+   * Global meta is provided with a lodash-style dot-separated path,
+   * and the value you wish to set:
+   * ```ts
+   * loggingSystem.setGlobalMeta('service.node.roles', ['ui']);
+   * // adds { service: { node: { roles: ['ui'] } } } to every log entry
+   * ```
+   * @param path Lodash-style dot-separated path to update in the LogMeta.
+   * @param meta Global meta to set at the path.
+   */
+  public setGlobalMeta(path: string, meta: unknown) {
```

Comment:
_insert speech about how naming is the worst and I'd be open to any better suggestions_ ❤️

Reply:
FWIW The log4j term for what we call 'meta' is `MDC` (Mapped Diagnostic Context)

Maybe `setGlobalContext`? Not that it's really better or imaginative than `setGlobalMeta`

Reply:
Yeah, I guess we're basically implementing MDC here, I'll go with `setGlobalContext`.